docs/fastdds/dds_layer/core/policy/standardQosPolicies.rst

```diff
@@ -1321,7 +1321,7 @@ List of QoS Policy data members:
 +------------------------------------------------------------+----------------------------------------+----------------+
 
 .. note::
-    This QoS Policy concerns to Topic entities.
+    This QoS Policy concerns to |Topic| entities.
```

Comment:
It's not obvious to me when you add the links. Are you trying to add them to _all_ Topics, DataWriters, and DataReaders? If so, there are some missing. If not, then why just to these ones?

Reply:
The idea was to add them to every note detailing which entities each QoS applies to. I have added them in the missing entities of xtypesExtensions file.